the-collab-lab/tcl-65-smart-shopping-list
# Issue #1 | Render names of items in shopping list

## Description

This PR adds the ListItem component to List.jsx to render the names of items on the list. Update Firebase config in config.js to connect DB). 

## Related Issue

Closes #1 

## Acceptance Criteria

 The streamListItems function is used in App.jsx to get the items in the current list from the Firestore database
 The list is passed into the List component as a prop named data
 In List.jsx, the ListItem component is used to render the name of each item

## Type of Changes


|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Updates

### Before
![Screen Shot 2023-08-16 at 12 14 22 PM](https://github.com/the-collab-lab/tcl-65-smart-shopping-list/assets/86745260/64e89294-8936-4620-93ae-f753366c21fd)



### After

https://github.com/the-collab-lab/tcl-65-smart-shopping-list/assets/86745260/00d8fd95-2623-4b0e-b51a-2670a313929e



## Testing Steps / QA Criteria

-From your terminal, pull down this branch with git pull origin is-sb-rendering-list-item-names and check that branch out with git checkout is-sb-rendering-list-item-names.
-run application using command npm start.
-add /list to URL to render updated list names (localhost:3000/list).
